brave/browser-laptop
# Add tor loading progress UI

Fix https://github.com/brave/browser-laptop/issues/14043

## Test Plan:

Success case:
1. start brave
2. immediately open a tor private tab
3. you should briefly see a loading message in the URLbar
4. once loading is finished, you should be able to type in the URLbar

Fail case:
1. start an HTTP listener on port 9250 (so that Tor cannot bind to it.
   this causes Tor to not be able to connect).
2. start brave
3. open tor tab and wait for 20s
4. you should see an error message pop up
5. the button and link in the error message should work as expected

![screen shot 2018-06-12 at 4 40 08 pm](https://user-images.githubusercontent.com/549654/41323088-4df18d1a-6e61-11e8-95a6-06a528d37485.png)


## Submitter Checklist:



## Reviewer Checklist:


Tests
